.yarnrc.yml

```diff
@@ -10,6 +10,6 @@ nodeLinker: node-modules
 
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-allow-scripts.cjs
-    spec: "https://raw.githubusercontent.com/LavaMoat/LavaMoat/main/packages/yarn-plugin-allow-scripts/bundles/@yarnpkg/plugin-allow-scripts.js"
+    spec: 'https://raw.githubusercontent.com/LavaMoat/LavaMoat/main/packages/yarn-plugin-allow-scripts/bundles/@yarnpkg/plugin-allow-scripts.js'
```

Comment:
A change to this file would seem to indicate that we haven't configured Prettier to ignore this file. I just checked and we do seem to be making an exception for this file (meaning that it should not be getting linted). Would you mind reverting this change? It would get blown away anyway if we added a new Yarn plugin or otherwised updated the Yarn config in any way.

Reply:
I have added a `prettierignore` file and reverted the changes

[b618965](https://github.com/MetaMask/KeyringController/pull/202/commits/b6189656984c6dcdaf7e511204038fb92beccb5b)